crates/jiter/src/parse.rs

```diff
@@ -67,9 +68,12 @@ impl<'j> Parser<'j> {
     pub fn new(data: &'j [u8]) -> Self {
         Self { data, index: 0 }
     }
-}
 
-impl<'j> Parser<'j> {
+    #[allow(dead_code)]
```

Comment:
Why the allow?

Reply:
because it's only used when the `python` feature is enabled